pages/orders.js

```diff
@@ -0,0 +1,135 @@
+import Layout from '../components/Layout'
+import useSWR from 'swr'
+import cookie from 'js-cookie';
+
+const fetcher = (url,token) => fetch(url,{headers: {'Content-Type': "application/json", 'Authorization': token}}).then((res) => res.json())
+
+const token = cookie.get("firebaseToken")
+
+class PackingOrders extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+          orders: props.data      
+        }
+    }
+
+    // Permanently deletes order
+    deleteOrder(orderId) {
```

Comment:
Could we ask to confirm before they delete an order? It feels like the button could be very dangerous

Reply:
Yeah good idea! Added a confirmation popup:

<img width="557" alt="Screen Shot 2022-04-07 at 11 10 17 PM" src="https://user-images.githubusercontent.com/20667332/162376000-559d2647-3090-443f-859c-446f3fa1b1f9.png">